src/xtgeo/grid3d/grid.py

```diff
@@ -778,7 +780,7 @@ def generate_hash(
 
         gid = "".join(str(getattr(self, att)) for att in required)
 
-        return xtgeo.common.sys.generic_hash(gid, hashmethod=hashmethod)
+        return generic_hash(gid, hashmethod=hashmethod)
```

Comment:
This is tagging along because the test began to fail due to not finding `sys` in `xtgeo.common` for unknown reasons